wasm-rust-sample/README.md

```diff
@@ -0,0 +1,76 @@
+# WebAssembly and Rust Example
+
+This plugin is a starting point for leveraging Rust and WebAssembly in your UXP plugins. It comes defined with most of the dependencies you need to get started. As this plugin does rely on the [Rust Programming Language](https://www.rust-lang.org/), an environment configured for Rust development will be required before this will be usable in XD.
+
+**Warning:** This plugin currently crashes XD on ARM devices, including Apple Silicon Macs.
```

Comment:
Decided to leave [this warning](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/pull/42) in for now, let me know if the XD version runs okay with ARM!